calico/about/product-comparison.mdx

```diff
@@ -0,0 +1,87 @@
+---
+description: Describes Tigera products, including a feature comparison table.
+---
+
+# Product comparison
+
+**{{prodname}} Open Source** is the base product that comprises both {{prodname}} Enterprise and {{prodname}} Cloud. It provides the core networking and network policy features.
+
+**{{prodname}} Enterprise** adds a web UI for visibility and troubleshooting Kubernetes clusters, and other value-added features on top of {{prodname}} Open Source. 
+
+**{{prodname}} Cloud** is the SaaS version of {{prodname}} Enterprise. It includes Image Assurance to scan and detect vulnerabilities in images, and container threat defense to detect malware. It also adds onboarding tutorials, and eliminates the cost to manage Elasticsearch logs and storage. 
```

Comment:
I don't think we want to position CC as a variant of Enterprise - they have different features and are different products. In general for these bullet points I think we don't want to define each product so much as a variation on another product.

Reply:
Reworded.

Reply:
I think it helps to understand the product differences. I'll let PM decide this one.